cmake/modules/FindLUG.cmake

```diff
@@ -0,0 +1,179 @@
+# This script locates the Lugdunum library
+# ------------------------------------
+#
+# Usage
+# -----
+#
+# When you try to locate the Lugdunum libraries, you must specify which modules you want to use (system, window, graphics, core, etc).
+# If none is given, the LUG_LIBRARIES variable will be empty and you'll end up linking to nothing.
+# example:
+#   find_package(LUG COMPONENTS graphics window system) # find the graphics, window and system modules
+#
+# You can't enforce a specific version yet.
+#
+# By default, the dynamic libraries of Lugdunum will be found. To find the static ones instead,
+# you must set the LUG_STATIC_LIBRARIES variable to TRUE before calling find_package(LUG ...).
+#
+# In case of static linking, the LUG_STATIC macro will also be defined by this script.
+# example:
+#   set(LUG_STATIC_LIBRARIES TRUE)
+#   find_package(LUG COMPONENTS system)
+#
+# If Lugdunum is not installed in a standard path, you can use the LUG_ROOT CMake (or environment) variable
+# to tell CMake where Lugdunum is.
+#
+# Output
+# ------
+#
+# This script defines the following variables:
+# - For each specified module XXX (system, window, graphics, core, etc):
+#   - LUG_XXX_LIBRARY_DEBUG:   the name of the debug library of the xxx module (set to LUG_XXX_LIBRARY_RELEASE is no debug version is found)
```

Comment:
This is the attended case, the lib which is called "system" will be named "SYSTEM" in the variable names.